packages/runtime-api/identities.js

```diff
@@ -114,6 +114,16 @@ class IdentitiesApi
     return !_.isEqual(memberId.raw, new Null());
   }
 
+  /*
+   * Return true if the account is an actor/role account
+   */
+  async isActor(accountId)
```

Comment:
There is a slightly better placed function in the `roles` API, `checkForRole` - or are you asking about *any* role, not a specific one?

It would make sense that *any* actor should be able to publish... but I suspect I'd still put it in the `roles` API, no?